internal/lang/funcs/descriptions.go

```diff
@@ -91,6 +91,13 @@ var DescriptionList = map[string]descriptionEntry{
 			"The maximum length of each chunk. All but the last element of the result is guaranteed to be of exactly this size.",
 		},
 	},
+	"cidrcontains": {
+		Description: "`cidrcontains` determines whether a given IP address or an address prefix given in CIDR notation is within a given IP network address prefix.",
+		ParamDescription: []string{
+			"`prefix` must be given in CIDR notation, as defined in [RFC 4632 section 3.1](https://tools.ietf.org/html/rfc4632#section-3.1).",
+			"`ip_or_prefix` is either an IP address or an address prefix given in CIDR notation.",
+		},
```

Comment:
not a technical issue, but a naming issue (and yes, naming is hard).

The parameters of "prefix" and "ip_or_prefix" work because they contrast.  But I could see some potential confusion if someone is testing prefixes and mistakenly reverses the order.

"parent prefix" and "ip or child prefix" seems better conceptually, but they're awfully annoying to write.  Your code uses containing and contained, so containing_prefix and ip_or_contained_prefix aligns to it, but again, awfully annoying to write out in full.

In the end, I think the current terms work fine, but I mention it just in case there are better terms that can be used.

Reply:
> awfully annoying to write out in full

To be fair, you just put it in the description, you never need to write it in the code, and the caller does not care. There's no good reason *not* to be verbose here if that helps the user.

Reply:
Then:  containing_prefix and contained_ip_or_prefix?  It's fairly descriptive and it matches the nomenclature of the implementation.

Reply:
Done.